.github/workflows/windows_test_comment_trigger.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   windows_test_preparation:
     runs-on: ubuntu-latest
-    if: ${{ github.event.issue.pull_request && github.event.comment.body == '/start Windows testing' }}
+    if: ${{ github.event.issue.pull_request && (github.event.comment.body == '/start Windows testing' ||  github.event.comment.body == '/windu please')}}
```

Comment:
Is this change intentional?

Reply:
Yes, do you think this alternative is not acceptable?

Reply:
I think it's confusing. Pavel's suggestion below is more reasonable.